packages/cli/src/commands/generate/scaffold/templates/components/NameCell.tsx.template

```diff
@@ -16,7 +16,7 @@ export const Loading = () => <div>Loading...</div>
 export const Empty = () => <div>${singularPascalName} not found</div>
 
 export const Failure = ({ error }: CellFailureProps) => (
-  <div className="rw-cell-error">{error.message}</div>
+  <div className="rw-cell-error">{error?.message}</div>
```

Comment:
Have to say, and maybe this is "strict" but given that this component would be shown only if there is an error, seems odd that needs the `?` since isn't its presence why the component is being rendered.

Reply:
Yeah its a bit weird - but Apollo says these errors are optional, even when it errors out. That's probably because there's some sort of case where it doesn't return an error case